tests/usermetadata/test_metadata.py

```diff
@@ -47,26 +52,71 @@ async def test_that_usermetadata_recipe_works_as_expected():
     start_st()
 
     version = await Querier.get_instance().get_api_version()
-    if compare_version(version, '2.13.0') != '2.13.0':
-        # If the version less than 2.13.0, user metadata doesn't exist. So skip the test
+    if compare_version(version, '2.12.0') != '2.12.0':
+        # If the version less than 2.12.0, user metadata doesn't exist. So skip the test
         return
 
     TEST_USER_ID = "userId"
-    TEST_METADATA = {"role": "admin", "name": {"first": "John", "last": "Doe"}}
+    TEST_METADATA: Dict[str, Any] = {"role": "admin", "name": {"first": "John", "last": "Doe"}}
+
+    get_metadata_res = await get_user_metadata(TEST_USER_ID, {})
```

Comment:
You should not have to pass `{}` as user_context. If you see the other comments in this PR, the functions exposed to the user should have `user_context` as optional